runtime/src/accounts_db.rs

```diff
@@ -4767,6 +4891,18 @@ impl AccountsDb {
         true
     }
 
+    fn is_candidate_for_shrink(&self, store: Arc<AccountStorageEntry>) -> bool {
```

Comment:
Should be able to borrow here to not clone the Arc. `&Arc<AccountStorageEntry>`

Reply:
Will fix

Reply:
Fixed